client/sso.go

```diff
@@ -279,22 +280,17 @@ func (cli *Client) createSsoBasic(apiPath string, sso interface{}) error {
 	return nil
 }
 
-// GetRoleMappingSass - returns Aqua RoleMappingSaas
-func (cli *Client) GetRoleMappingSass(id string) (*RoleMappingSaas, error) {
+// GetRoleMappingSaas - returns Aqua RoleMappingSaas
+func (cli *Client) GetRoleMappingSaas(id string) (*RoleMappingSaas, error) {
 	var err error
 	var response RoleMappingSaas
 	baseUrl := ""
 	apiPath := fmt.Sprintf("/v2/samlmappings/%s", id)
-	switch cli.clientType {
-	case Csp:
-		err = fmt.Errorf("GetRoleMappingSass is Supported only in Aqua SASS env")
-		return nil, err
-	case SaasDev:
-		baseUrl = consts.SaasDevTokenUrl
-	case Saas:
-		baseUrl = consts.SaasTokenUrl
-	default:
-		err = fmt.Errorf("GetRoleMappingSass is Supported only in Aqua SASS env")
+
+	if cli.clientType == Saas || cli.clientType == SaasDev {
+		baseUrl = cli.tokenUrl
+	} else {
+		err = fmt.Errorf("GetRoleMappingSaas is Supported only in Aqua SaaS env")
 		return nil, err
 	}
```

Comment:
These codeblocks fixes the issue #244. They already existed in `group.go` and `user.go`